docs/src/tutorials/deim_FitzHugh-Nagumo.md

```diff
@@ -0,0 +1,189 @@
+nothing # hide
+```
+
+Next, we apply finite difference discretization using 
+[MethodOfLines.jl](https://github.com/SciML/MethodOfLines.jl).
+
+```@example deim_FitzHugh_Nagumo
+using MethodOfLines
+N = 15 # equidistant discretization intervals
+dx = (L - 0.0) / N
+dxs = [x => dx]
+order = 2
+discretization = MOLFiniteDifference(dxs, t; approx_order = order)
+ode_sys, tspan = symbolic_discretize(pde_sys, discretization)
+simp_sys = structural_simplify(ode_sys)
+ode_prob = ODEProblem(simp_sys, nothing, tspan)
+nothing # hide
+```
+
+The snapshot trajectories are obtained by solving the full-order system. 
+
+```@example deim_FitzHugh_Nagumo
+using DifferentialEquations
+sol = solve(ode_prob, Tsit5())
+sol_x = sol[x]
+nₓ = length(sol_x) # number of discretization points in x
+nₜ = length(sol[t]) # number of discretization points in time
+nothing # hide
+```
+
+Let's see the fast decay of the singular values of the snapshot solutions for ``v``, ``w``,
+and the nonlinear snapshots ``f(v)``.
+
+```@example deim_FitzHugh_Nagumo
+using LinearAlgebra
+snapshot_v = sol[v(x, t)]
+snapshot_w = sol[w(x, t)]
+snapshot_fv = f.(snapshot_v)
+svdval_v = svdvals(snapshot_v)
+svdval_w = svdvals(snapshot_w)
+svdval_fv = svdvals(snapshot_fv)
+using Plots, LaTeXStrings
+svd_plt = plot(yscale = :log10, xticks = eachindex(svdval_v), titlefont = 11,
+               legendfont = 10, title = "Singular values of the snapshots")
+plot!(svd_plt, svdval_v, markershape = :circle, label = L"Singular Val of $v$")
+plot!(svd_plt, svdval_w, markershape = :circle, label = L"Singular Val of $w$")
+plot!(svd_plt, svdval_fv, markershape = :circle, label = L"Singular Val of $f(v)$")
+```
+
+The following figure shows the phase-space diagram of ``v`` and ``w`` at different spatial
+points ``x`` from the full-order system.
+
+```@example deim_FitzHugh_Nagumo
+full_plt = plot(xlabel = L"v(x,t)", ylabel = L"x", zlabel = L"w(x,t)", xlims = (-0.5, 2.0),
+                ylims = (0.0, L), zlims = (0.0, 0.25), legend = false, xflip = true,
+                camera = (50, 30), titlefont = 10,
+                title = "Phase−Space diagram of full $(nameof(pde_sys)) system")
+@views for i in 1:nₓ
+    plot!(full_plt, snapshot_v[i, :], _ -> sol_x[i], snapshot_w[i, :])
+end
+plot!(full_plt)
+```
+
+Then, we use POD to construct a linear subspace of dimension, say, 5 for the system's state
+space and project the model onto the subspace. DEIM is employed to approximate nonlinear 
+terms. This can be done by simply calling [`deim`](@ref).
+
+```@example deim_FitzHugh_Nagumo
+using ModelOrderReduction
+snapshot_simpsys = Array(sol.original_sol)
+pod_dim = deim_dim = 5
+deim_sys = deim(simp_sys, snapshot_simpsys, pod_dim)
+deim_prob = ODEProblem(deim_sys, nothing, tspan)
+deim_sol = solve(deim_prob, Tsit5())
+nₜ_deim = length(deim_sol[t])
+sol_deim_x = deim_sol[x]
+sol_deim_v = deim_sol[v(x, t)]
+sol_deim_w = deim_sol[w(x, t)]
```

Comment:
I see, so the original variables are written as observed variables of the DEIM system? I guess I'll see that further down. Nice that it works out. I wonder what the current scaling is like.